modules/utxo-lib/src/bitgo/psbt/fromHalfSigned.ts

```diff
@@ -47,14 +47,7 @@ export function getInputUpdate(tx: UtxoTransaction<bigint>, vin: number, prevOut
       throw new Error('Invalid scriptSig, failed to decompile');
     }
     if ((!input.witness || !input.witness.length) && decompiledScriptSig.length === 2) {
-      // P2PKH
-      const pubkey = decompiledScriptSig.pop();
-      const signature = decompiledScriptSig.pop();
-      if (!Buffer.isBuffer(pubkey) || !Buffer.isBuffer(signature)) {
-        throw new Error('Invalid pubkey or signature');
-      }
-      update.partialSig = [{ pubkey, signature }];
-      return update;
+      throw new Error();
```

Comment:
Should we have some more context on why this is throwing an emtpy error? Maybe message saying due to deprecation or some other reasoning behind this?

Reply:
I will get rid of this branch in the next PR